demo/index.html

```diff
@@ -459,6 +459,36 @@ <h2 id="setreturnfocus-heading">setReturnFocus option</h2>
     </div>
   </div>
 
+  <div id="demo-multipleelements">
```

Comment:
Thank you for adding a test case. This is great. I feel we need another one that tests this multi-container feature along with multiple traps where we're able to activate a second trap, having the first one "paused", and each trap is able to deal with its own set of multiple containers, just to flex that muscle a bit and make sure it all works as expected.

Reply:
Done.